soccer/Processor.cpp

```diff
@@ -85,14 +86,16 @@ Processor::Processor(bool sim, bool defendPlus, VisionChannel visionChannel)
     _gameplayModule = std::make_shared<Gameplay::GameplayModule>(&_state);
     _pathPlanner = std::unique_ptr<Planning::MultiRobotPathPlanner>(
         new Planning::IndependentMultiRobotPathPlanner());
-    vision.simulation = _simulation;
-
+    vision.simulation = true;
```

Comment:
I'm not sure, but won't this break soccer on real robots? I'm not sure exactly what this variable does though...